firmware/src/rdm_responder.c

```diff
@@ -192,6 +199,89 @@ int RDMResponder_GenericReturnString(const RDMHeader *header,
   return RDMUtil_AppendChecksum(g_rdm_buffer);
 }
 
+int RDMResponder_GenericGetBool(const RDMHeader *header, bool value) {
```

Comment:
Well that's different because MODEL_ID has bounds checking. The generic functions are only used if there is no bounds checking.